core/services/ocr2/plugins/directrequestocr/scripts/gen_oracle_contract_config.go

```diff
@@ -147,6 +157,19 @@ func main() {
 		transmittersStr = append(transmittersStr, string(transmitters[i]))
 	}
 
+	config := OCR2DROracleConfig{
+		Signers:               signersStr,
+		Transmitters:          transmittersStr,
+		F:                     f,
+		OnchainConfig:         "",
+		OffchainConfigVersion: offchainConfigVersion,
+		OffchainConfig:        "0x" + hex.EncodeToString(offchainConfig),
+	}
+
+	file, _ := json.MarshalIndent(config, "", " ")
+
+	_ = ioutil.WriteFile("OCR2DROracleConfig.json", file, 0644)
```

Comment:
Generally a bad idea to ignore errors. Always check for errors and at least print them to the console/log to see what's wrong.